source/elements/oneDAL/source/terminology.rst

```diff
@@ -0,0 +1,64 @@
+.. _terminology:
+
+============
+Terminology
+============
+
+@table-begin:
```

Comment:
I expect you'll convert that into an rst table?

Reply:
converted to rst table